system/security/playlist.xml

```diff
@@ -27,6 +27,16 @@
 	To avoid test target duplication, this belongs to sanity, extended and special. Regular test should only belong to one level -->
 	<test>
 		<testCaseName>OAuthTest</testCaseName>
+		<disables>
+			<disable>
+				<comment>Temporarily disabled until following is fixed: eclipse-openj9/openj9/issues/13892</comment>
```

Comment:
This is an open issue. Could you please use the URL link? (no other comment is needed here)
We will use the information for the disabled test pipeline in the future. Thanks

Reply:
Sure. Updated.

Reply:
Thanks @Mesbah-Alam